application/comit_node/src/swap_protocols/rfc003/bob/actions/btc_erc20.rs

```diff
@@ -127,12 +127,18 @@ impl Actions for bob::State<Bitcoin, Ethereum, BitcoinQuantity, Erc20Token> {
                     secret,
                 ))]
             }
+
             (Funded { .. }, NotDeployed, _) => {
                 vec![bob::ActionKind::Deploy(deploy_action(&request, &response))]
             }
+
             (Funded { .. }, Deployed { htlc_location, .. }, _) => vec![bob::ActionKind::Fund(
                 fund_action(&request, &response, *htlc_location),
             )],
+            (_, Funded { htlc_location, .. }, _) => vec![bob::ActionKind::Refund(refund_action(
```

Comment:
❌  This doesn't show the refund action `alpha_ledger` is in `funded` state. It should always return refund when beta is in funded state.